pov_tools/create_POV_data_files

```diff
@@ -0,0 +1,251 @@
+#!/bin/bash
+
+# create_POV_data_files
+
+OIFS="$IFS"
+IFS=$'\n'
+cd "$(dirname "$0")"
+clear
+height_only=0
+preset=0
+pngcount=0
+wscount=0
+rename=0
+remove=0
+
+echo ""
+echo "Welcome! Let's make some POV images!"
+echo ""
+echo "Found these image files:"
+# get file names and quantity of existing .png files
+# check for whitespace in name
+for pngs in `find . -maxdepth 1 -type f -name "*.png"`; do
```

Comment:
The initial find might be multiples. Later it could become singular png, sure. 
If we're sticking with the way it has worked up until now (processing one file only).
Questions as to "why" no multiples or spaces or 'preview' contained in the name are all answered with: It's just conventions to make the automation work the way doing a manual make did. If we're changing how many files can be processed and are allowed etc.. that of course changes the course of this script.